### PR TITLE
day2, ghibli API

### PR DESCRIPTION
**Pull request by cal1co**
- How difficult did you find this (out of 10)? (0 being no problems at all, 10 being impossible)
3.

- Was there anything that you struggled with?
selecting/searching with the api 

- Is there any particular code you want me to look into?
- Is there anything that you'd like some further information on?
What's the best way to output an object of objects into an array in a sorted order (by a value of their key)? I wanted to sort the movies by their rt_rating then show the top 10. The movies I got weren't sorted properly though.

- Roughly how long did it take?
1.5hr

ty, Ro!

### CODE REVIEW

File: Alex King/WK7HW/DAY2/ghibli-search/js/main.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        axios.get(url)
+            .then( (res) => {
+                function findFavs(title){
+                    res.data.forEach((e)=>{
+                        if (e.title.includes(title)){
+                            renderSearchResults(e)
+                        } 
+                    }) // forEach
+                }
+                favourites = ["Porco Rosso", "Howl's Moving Castle", "Ponyo", "Kiki's Delivery Service", "Spirited Away"]
```
Comment (roweeena) [inline Alex King/WK7HW/DAY2/ghibli-search/js/main.js:19]: you don't like Princess Mononoke?!

**Comment by roweeena** [comment]
Are you asking so you can render all the films sorted out by rating on your home page? Also found a small bug when you click on the word "home" (not the home button) it tries to fetch a movie

File: Alex King/WK7HW/DAY2/ghibli-search/js/main.js (hunk 1 above)
Comment (cal1co) [reply Alex King/WK7HW/DAY2/ghibli-search/js/main.js:19]: I love Mononoke but if I had more than 5, the page looked weird. I had to make some sacrifices! 😵

**Comment by cal1co** [comment]
> Are you asking so you can render all the films sorted out by rating on your home page? Also found a small bug when you click on the word "home" (not the home button) it tries to fetch a movie

Yeah, I want to have a sorted output of movies.
I only realised the bug late last night. It's because my on click is on the wrong div. It happens on the show page for each movie also 😭

**Comment by roweeena** [comment]
you need to convert the scores to numbers and then do something like 
```
function compare( a, b ) {

  if ( a.rt_score < b.rt_score ){
    return -1;
  }
  if ( a.rt_score > b.rt_score ){
    return 1;
  }
  return 0;
}

res.data.sort( compare );
```

which will get back the objects sorted by score and then you can loop through the array to show them. does that help?

**Comment by cal1co** [comment]
Omg the numbers are strings. I did almost exactly that but it wasn't doing anything because they weren't integers 😭. tysm!!

**Pull request merged**